### PR TITLE
OSDOCS-10835: adds certificate cleaning option in relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.16

Issue:
[OSDOCS-10835](https://issues.redhat.com/browse/OSDOCS-10835)

Link to docs preview:
[Cleaning up the certificates are supported](https://77217--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-configuring)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:
[Main PR for reference](https://github.com/openshift/openshift-docs/pull/77156)

### CODE REVIEW

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -63,6 +63,10 @@ With this release, you can configure a custom server certificate that has been i
 
 You can now configure audit logging policies to manage the retention policies for log files, ensuring that edge devices with limited storage capacities are not hampered by accumulated logging data. To configure audit log policies, use settings such as a maximum file size limit and maximum retained files to set a limit on log storage size. You can also choose an audit policy profile to specify the data collected. See xref:../microshift_configuring/microshift-audit-logs-config.adoc#microshift-audit-logs-config[Configuring audit logs].
 
+[id="microshift-4-16-certificates-cleaning"]
+==== Cleaning up the certificates are supported
```
Comment (bergerhoffer) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:67]: This sounded a little awkward (especially "are"), making another suggestion
```suggestion
==== Support for cleaning up certificates
```

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 2]
```diff
@@ -63,6 +63,10 @@ With this release, you can configure a custom server certificate that has been i
...
+With this release, you can clean up the custom certificates. See xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca-certificates-cleaning_microshift-custom-ca[Cleaning up and recreating the custom certificates] for details.
```
Comment (bergerhoffer) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:68]: We typically recommend the following standard format of "For more information, see <link>"

```suggestion
With this release, you can clean up custom certificates. For more information, see xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca-certificates-cleaning_microshift-custom-ca[Cleaning up and recreating the custom certificates].
```

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-16-release-notes.adoc]: done

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 2 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-16-release-notes.adoc]: done

**Pull request merged**